nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -291,58 +291,62 @@ in
       };
       defaultNotePath = mkOption {
         type = types.nullOr types.str;
-        default = "./public/default.md";
+        default = "${cfg.package}/public/default.md";
+        defaultText = "$''{cfg.package}/public/default.md";
```

Comment:
```suggestion
        defaultText = literalExpression "\${cfg.package}/public/default.md";
```

Same for the other options

Reply:
Wait no, sorry

```suggestion
        defaultText = literalExpression ''"''${cfg.package}/public/default.md"'';
```

Reply:
Lets take the second with the escaped ``\"`` because ``''"''`` is a bit ugly.